snowshu/core/main.py

```diff
@@ -102,15 +106,18 @@ def create(replica_file: click.Path,
         exists=True),
     default=REPLICA_DEFAULT,
     help="where snowshu will look for your replica configuration file, default is ./replica.yml")
+@click.option('--target',
```

Comment:
There's no need to include an override option to the CLI. We should only use the override in the case of an incremental build, which will be taken care of by a future `--incremental <target_image>` option